### PR TITLE
Add refinement to 'draing' typo

### PR DESCRIPTION
**Pull request by luzpaz**
Added 'draining' as alternative solution. Example in the wild:  
https://github.com/gphoto/libgphoto2/blob/0b93f2d0d5305e941017d2d7cc6e04952261e126/camlibs/ptp2/library.c#L5662

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -20177,7 +20177,7 @@ draconain->draconian
 dragable->draggable
 draged->dragged
 draging->dragging
-draing->drawing
+draing->drawing, draining,
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:20180]: Indeed, but in this example I think the intended word was `drain`, not `draining`.
```suggestion
draing->drawing, drain, draining,
```

Comment (luzpaz) [reply codespell_lib/data/dictionary.txt:20180]: Screenshot_20230902_104053

Can't seem to commit via GH interface

Comment (luzpaz) [reply codespell_lib/data/dictionary.txt:20180]: Added manually in adb7ec0

Comment (DimitriPapadopoulos) [reply codespell_lib/data/dictionary.txt:20180]: Yes, the GitHub interface is broken, it cannot modify files > 1 MB.

Perhaps it's time to split dictionary.txt into multiple files.

**Review by DimitriPapadopoulos — APPROVED** [review]

**Pull request merged**